src/App.js

```diff
@@ -1,26 +1,25 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react"
+import "./App.css"
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import {
+  BrowserRouter as Router,
+  Route,
+} from "react-router-dom";
+
+import Home from './pages/Home';
+
+class App extends Component {
+  render() {
+    return (
+      < Router >
+        <Route exact path="/">
+          <div className="App">
```

Comment:
Is this div being referenced anywhere? If not I think we can delete the div tag here

Reply:
Ok got it deleted the div tag here in App.js and used the className="App" to the div tag in the Home.js to keep the center alignment of text